kamailio/default.cfg

```diff
@@ -842,4 +872,4 @@ event_route[tm:local-request]
 #!endif
 
 
-# vim: tabstop=4 softtabstop=4 shiftwidth=4 expandtab
+# vim: tabstop=4 softtabstop=4 shiftwidth=4 expandtab
```

Comment:
It snuck in by accident. Not sure how but was not intended.
Is it a standard to always have a newline at the end of all files or just config files?